src/main/java/org/spongepowered/api/data/key/Keys.java

```diff
@@ -1458,6 +1459,44 @@
      */
     public static final Key<Value<LogAxis>> LOG_AXIS = DummyObjectProvider.createExtendedFor(Key.class,"LOG_AXIS");
 
+    /**
+     * Represents the {@link Key} for the world for a
+     * {@link ItemTypes#FILLED_MAP}
+     *
+     * @see MapItemData#world()
+     */
+    public static final Key<Value<World>> MAP_WORLD = DummyObjectProvider.createExtendedFor(Key.class, "DIMENSION");
+
+    /**
+     * Represents the {@link Key} for the centre x and z of where a
+     * {@link ItemTypes#FILLED_MAP} represents
+     *
+     * @see MapItemData#location()
+     */
+    public static final Key<Value<Vector2i>> MAP_LOCATION = DummyObjectProvider.createExtendedFor(Key.class, "MAP_LOCATION");
+
+    /**
+     * Represents the {@link Key} for whether a map tracks player
+     *
+     * @see MapItemData#trackPosition()
+     */
+    public static final Key<Value<Boolean>> MAP_TRACKS_PLAYERS = DummyObjectProvider.createExtendedFor(Key.class, "MAP_TRACKS_PLAYERS");
+
+    /**
+     * Represents the {@link Key} for whether a map can track
+     * a player from anywhere in the world
+     *
+     * @see MapItemData#unlimitedTracking()
+     */
+    public static final Key<Value<Boolean>> MAP_UNLIMITED_TRACKING = DummyObjectProvider.createExtendedFor(Key.class, "MAP_UNLIMITED_TRACKING");
+
+    /**
+     * Represents the {@link Key} for the scale of a map
+     *
+     * @see MapItemData#scale()
+     */
+    public static final Key<Value<Byte>> MAP_SCALE = DummyObjectProvider.createExtendedFor(Key.class, "MAP_SCALE");
```

Comment:
Use `Integer`

Reply:
minecraft's scale field in mapdata is only byte, so it could cause problems if we allowed more than a byte's range

Reply:
Then you can make it a bounded value, which will restrict the min/max values: `Key<BoundedValue<Integer>>`
Using a byte leaks implementation details that shouldn't be in the API, this also prevent future changes and it's more conventient to work with integers.